src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, serif;
+  font-weight: 300;
+}
+
+body {
+  margin-top: 20px;
+}
+
+.input {
+  background: #fff;
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.input-small {
+  height: 42px;
+  padding-left: 33px;
+}
+
+.input::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.input:focus::placeholder {
+  font-weight: 800;
+}
+
+.input-small::placeholder {
+  font-size: 14px;
+}
+
+.input-form {
+  position: relative;
+}
+
+.search-icon::before {
```

Comment:
create a common class to prevent code duplicating and for one of images to set differences

Reply:
how to set differences if the class is common?